src/path_converters.h

```diff
@@ -363,10 +366,30 @@ class PathClipper : public EmbeddedQueue<3>
                     goto exit_loop;
 
                 case agg::path_cmd_move_to:
+
+		    // was the last command a move to (and we have
+		    // seen at least one command ?
+		    // if so, shove it in the queue if in clip box
+		    if (m_moveto && m_has_init &&
+			m_lastX >= m_cliprect.x1 &&
```

Comment:
I prefer reversing the arguments here so that it looks similar to Python's chained form.